test-worker/test/asset-compute/worker/post-skip-jpg-quality/params.json

```diff
@@ -0,0 +1,5 @@
+{
+    "imagemagick": "-quality 10",
```

Comment:
I thought we didn't want to expose command line args?

Reply:
This is only implemented by the test worker to quickly simulate a worker tool "already" doing something (like fmt or resize): https://github.com/adobe/asset-compute-sdk/blob/b9b6e00f777895f70937e721df9a04bd1f6e2e54/test-worker/worker.js#L26